apps/next-react-native/src/pages/nft/[chainName]/[contractAddress]/[tokenId]/index.tsx

```diff
@@ -7,25 +7,27 @@ import { getMediaUrl } from "app/utilities";
 
 export async function getServerSideProps(context) {
   const { chainName, contractAddress, tokenId } = context.params;
-  const res = await axios.get(
-    `${process.env.NEXT_PUBLIC_BACKEND_URL}/v2/token/${contractAddress}/${tokenId}?chain_identifier=${CHAIN_IDENTIFIERS[chainName]}`
-  );
+  try {
+    const res = await axios.get(
+      `${process.env.NEXT_PUBLIC_BACKEND_URL}/v2/token/${contractAddress}/${tokenId}?chain_identifier=${CHAIN_IDENTIFIERS[chainName]}`
+    );
 
-  const nft = res.data?.data?.item as NFT;
-  const imageUrl = getMediaUrl({ nft, stillPreview: false });
-  if (nft) {
-    return {
-      props: {
-        meta: {
-          title: nft.token_name,
-          description: nft.token_description,
-          image: imageUrl,
+    const nft = res.data?.data?.item as NFT;
+    const imageUrl = getMediaUrl({ nft, stillPreview: false });
+    if (nft) {
+      return {
+        props: {
+          meta: {
+            title: nft.token_name,
+            description: nft.token_description,
+            image: imageUrl,
+          },
         },
-      },
-    };
-  }
+      };
+    }
+  } catch (e) {}
```

Comment:
need to improve error handling here, didn't want to spend too much time as we're using it just for metatags for now. will revisit, we can hydrate this data in swr.